util/fileUtils.go

```diff
@@ -159,12 +160,23 @@ func IsDir(path string) bool {
 
 // GetSpecFiles returns the list of spec files present at the given path.
 // If the path itself represents a spec file, it returns the same.
+var exitWithMessage = func(message string) {
+	logger.Errorf(true, message)
+	os.Exit(1)
```

Comment:
Can we use `logger.Fatal` here?

Reply:
it prints some extra information 
```
Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://spectrum.chat/gauge

Your Environment Information -----------
```
I don't think we need this as the error message itself explains the issue.

Reply:
Ok